docs/farming-&-staking/farming/intro.mdx

```diff
@@ -0,0 +1,145 @@
+This file system maximizes usable space for the farmer.
+```bash title="Recommended Formatting Command for Drives Dedicated to Subspace Plots:"
+sudo mkfs.ext4 -m 0 -T largefile4 /dev/sdX
+```
+</TabItem>
+
+<TabItem value="xfs" label="❌ XFS">
+Resizing a plot when the drive is nearly full (98% or more) will render the plot unfarmable unless sufficient additional space is freed up through other means. Full disk utilization can only be achieved during the plot's initial creation and will yield less usable space compared to ext4.
+</TabItem>
+
+<TabItem value="cow" label="❌ CoW">
+Copy-on-Write file systems (ReFS, BTRFS, ZFS, bcachefs, HAMMER) are not recommended due to increased latency and reduced write speeds associated with managing unnecessary data copies.
+```bash title="Disable CoW on BTRFS:"
+sudo chattr +C /path/to/farm
+```
+</TabItem>
+
+<TabItem value="raid" label="❌ RAID">
+The farmer employs checksums for data integrity and can recover from corrupted blocks. Striping provides no benefits and raises the risk of needing to replot multiple drives instead of just one.
+</TabItem>
+
+<TabItem value="swap" label="❌ SWAP">
+Only necessary if your system has limited RAM.
+</TabItem>
+
+</Tabs>
+
+## Security Considerations
+
+:::info Safety and Security Guide
+
+For detailed information, visit our [Safety and Security](../../learn/security) guide.
+
+:::
+
+
+To establish a secure farming environment, ensure your system is regularly updated, configure [port forwarding](additional-guides/port-config.mdx#ports-to-forward--firewall-access) and [configure your firewall](additional-guides/port-config.mdx#firewall-configuration), and adhere to established network safety protocols. For in depth security practices, please consult our [Safety and Security](../../learn/security) guide.
+
+Ensure your system is regularly updated, configure [port forwarding](additional-guides/port-config.mdx#ports-to-forward--firewall-access), set up your [firewall](additional-guides/port-config.mdx#firewall-configuration), and follow best practices for network safety. For comprehensive security guidelines, please refer to our [Safety and Security](../../learn/security) guide.
+
+
+
+## Wallet
+
+:::info Wallets Guide
+
+For detailed information, visit our [Wallets](../../category/wallets) guide.
+
+
+:::
+
+Before you begin, make sure you have a wallet to receive rewards. We recommend using [SubWallet](https://subwallet.app), but [PolkadotJS](https://polkadot.js.org/extension) is also available. Install one of these wallets as a browser extension and create a new account. Your account address will be required during the setup process.
+
+## Networking
+
+:::info Port Forwarding & Firewall Guide
+
+For detailed information on network configurations, including port forwarding requirements and firewall settings, please refer to our [Port Forwarding & Firewall](../../farming-&-staking/farming/additional-guides/port-config.mdx) guide.
+
+:::
```

Comment:
Let's re-add the Dependencies section from the original getting started quite, as far as I know all information inside of it is still relevant

```suggestion
:::

### Dependencies

:::caution  Windows No Output Bug
If you face an error where the node outputs nothing and no error code is given it is likely you just need to install the latest Visual C++ Redistributable package [here](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170)
:::

:::caution Linux `libcomp.so.1` error.
If you encounter an error related to `libgomp.so.1`, install the `libgomp1` library with `sudo apt-get install libgomp1`.
:::

:::caution Linux File Descriptor Limit
Linux has a hard limit of 100k file descriptors. Exceeding this may cause errors. Refer to our guide [here](./advanced-cli/cli-tips.mdx#changing-number-of-open-files-limit-linux)
:::
```